src/main/java/com/nike/cerberus/domain/input/ManagementServiceRegionSpecificInput.java

```diff
@@ -16,57 +16,18 @@
 
 package com.nike.cerberus.domain.input;
 
-import java.util.List;
-
-
 /**
- * Stores Management Service-specific parameters parsed from YAML
+ * Stores region specific Management Service parameters parsed from YAML
  */
-public class ManagementService {
+public class ManagementServiceRegionSpecificInput {
```

Comment:
nit: same comment about the `region` language in the input class names. I'm not sure what it adds?

It is more confusing IMO.

Reply:
There is already a class called ManagementServiceInput that has global properties

Reply:
Is there any reason to separate region-specific from non-region-specific?

The input classes seems to be a POJOs that don't care where their populated values come from (region-specific or not).

Reply:
It would no longer represent the YAML and would allow people to create invalid YAML configurations that wouldn't work and would be much harder to validate.